portafly/src/pages/Overview.tsx

```diff
@@ -1,32 +1,90 @@
-import React from 'react'
+import React, { useState, useContext } from 'react'
 import {useA11yRouteChange, useDocumentTitle} from 'components'
 import {
   PageSection,
   TextContent,
   Title,
   Text,
   Card,
-  CardBody
+  CardBody,
+  Modal,
+  Button
 } from '@patternfly/react-core'
 
+import moment from 'moment'
+import { I18nContext } from 'i18n/I18nProvider'
+import { 
+  OVERVIEW as STRINGS_ENG,
+  COMMONS as COMMONS_ENG
+} from 'i18n/locales/en/strings'
+import { 
+  OVERVIEW as STRINGS_JAP,
+  COMMONS as COMMONS_JAP
+} from 'i18n/locales/jp/strings'
+
 const Overview: React.FunctionComponent = ({children}) => {
+  const [strings, setStrings] = useState(STRINGS_ENG)
+  const [commonStrings, setCommonStrings] = useState(COMMONS_ENG)
+  const [locale, setLocale] = useState('en')
+  const polyglot = useContext(I18nContext)
```

Comment:
Question: is it necessary to wrap everything in a Context?

Reply:
Not necessary. You can import polyglot directly and create an instance here. But the idea is to use context instead of creating a lot of instances in several files